lib/soapy_cake/client.rb

```diff
@@ -22,7 +22,7 @@ def xml_response?
     attr_reader :domain, :api_key, :time_converter, :opts, :logger
 
     def run(request)
```

Comment:
Hmm, cake-issues will not be needed anymore after tomorrow (?) when the bucket_cake PR is merged.